django_jsx/management/commands/compilejsx.py

```diff
@@ -29,10 +29,12 @@
         function(el) {
             let ctx = JSON.parse(el.dataset.ctx)
             let component = jsx_registry[el.dataset.sha1](COMPONENTS, ctx)
-            ReactDOM.render(component, el)
-            if (el.parentNode) {
-                el.parentNode.replaceChild(el.children[0], el)
-            }
+            el.insertAdjacentHTML("afterend", "<span></span>")
```

Comment:
Could you add a comment explaining this code? Maybe something like '// Render the React component, replacing the django-jsx script tag' (if my interpretation is correct, that is...)